src/thor/route_action.cc

```diff
@@ -262,6 +263,8 @@ thor::PathAlgorithm* thor_worker_t::get_path_algorithm(const std::string& routet
     PointLL ll2(destination.ll().lng(), destination.ll().lat());
     if (ll1.Distance(ll2) < max_timedep_distance) {
       return &timedep_forward;
+    } else {
+      add_warning(request, 402);
```

Comment:
further down we could still choose unidir a* if it's trivial/connected pair, but in that case it hardly matters if a time was passed in